starcli/search.py

```diff
@@ -158,13 +165,22 @@ def search_by_spoken_language(
             if anchor_list[1].text
             else -1
```

Comment:
I think we should make all of these (and line 160) to None, like watchers count, so that in get_stats() we don't have to check if anything is -1

Reply:
https://github.com/hedythedev/starcli/blob/9199839bb3c70e37585eca2e356772ff29aac7a9/starcli/search.py#L179-L181

Here as you can see for ordering the repos, I have implemented the sorting by myself and it sorts based on `stargazers_count`. 

https://github.com/hedythedev/starcli/blob/9199839bb3c70e37585eca2e356772ff29aac7a9/starcli/search.py#L152-L153

Therefore, if I replace -1 with None, the sort would cause an error as it would fail to compare `None` to `int`. I can refactor for `forks_count` so that -1 would be replaced by None. Or I could keep them both as it is and improve the sorting by including the `forks_count` as well along with `stargazers_count`.

Reply:
Ok we should keep the sorting as is to stargazer count, but as to the -1 and None, we should keep star/fork/watch consistent so maybe make all of them 0 or -1 when the information can’t be scraped from the website

Reply:
I'll refactor it to -1 incase the information isn't scraped as 0 is still a valid number of stars.